command.go

```diff
@@ -9,15 +9,20 @@ import (
 )
 
 type Command struct {
-	Name     string
-	Interval time.Duration
-	Timeout  time.Duration
-	Exec     string
-	Params   []string
-	Enabled  bool
-	lastRun  time.Time
+	Name           string
+	Interval       time.Duration
+	Timeout        time.Duration
+	Exec           string
+	Params         []string
+	Enabled        bool
+	SecondsOverdue time.Duration
```

Comment:
SecondsOverdue must not be stored inside Command. The reason is because you never store something that you can compute, unless there is a performance reason for it (ie it is a cache). It may be slightly more painful to perform a sort, but it's worth it.